routes/web/billing_info.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CloverWeb
+  hash_branch("billing-info") do |r|
+    r.on String do |bi_ubid|
+      @billing_info = BillingInfo.from_ubid(bi_ubid)
```

Comment:
We need to check if this billing info is accessible by this user, no?

Reply:
Removed this path